presto-testing/src/main/java/io/prestosql/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -423,7 +423,11 @@ protected void assertPushedDown(String sql)
     protected void assertPushedDown(String actual, String expectedOnH2)
     {
         assertQuery(actual, expectedOnH2);
+        assertPushdedDownPlan(actual);
+    }
 
+    protected void assertPushdedDownPlan(String actual)
```

Comment:
Verifying results, besides plan, is an important thing.

Please rebase on top of https://github.com/prestosql/presto/pull/4466 and drop the change here.

Reply:
The approximate functions are not expected to match. I can either remove the approx test, just compare the plans or add an expected range. Let me know which one you prefer.

Reply:
Correct. Also, any floating point-bearing functions do not have to precisely match either.
We could use something like `org.testng.Assert#assertEquals(double, double, double)`.
Tempto has this built-in https://github.com/prestosql/tempto/blob/100163fb1d99dfde8c4b98fe1753e20d3ce9df56/tempto-core/src/main/java/io/prestosql/tempto/assertions/QueryResultValueComparator.java#L43
I think we should do something similar here.

Reply:
Added, take a look and let me know if you agree with the approach.